### PR TITLE
Make pagination respond to changing page better

### PR DESCRIPTION
**Pull request by jacobwindsor**
- Fixes issues in #58 
  - First and last page links not shown when on first or last page
  - UI fixes for changing the layout depending on the page

### CODE REVIEW

File: src/app/projects/components/pagination/pagination.component.ts [hunk 1]
```diff
@@ -48,6 +48,7 @@ export class PaginationComponent implements OnInit, OnChanges {
     ) {
       return;
     }
+    console.log(pageNumber);
```
Comment (MightyAx) [inline src/app/projects/components/pagination/pagination.component.ts]: should probably remove this.

**Review by MightyAx — APPROVED** [review]
This is so much better in use now.
I've removed a `console.log` that I presume was checked in accidentally.

**Review by aaclan-ebi — APPROVED** [review]
very noice!!!

**Comment by aaclan-ebi** [comment]
hold on, i think there's a bug when there's only 1 page

Screenshot 2021-06-24 at 11 56 11

**Comment by aaclan-ebi** [comment]
Screenshot 2021-06-24 at 11 58 20

**Review by MightyAx — CHANGES_REQUESTED** [review]
please fix bug for displaying less than 2 pages of results...
perhaps just hide the paginator?

File: src/app/projects/components/pagination/pagination.component.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<nav class="vf-pagination" aria-label="Pagination" *ngIf="totalPages < 2">
+<nav class="vf-pagination" aria-label="Pagination" *ngIf="totalPages <= 1">
```
Comment (MightyAx) [inline src/app/projects/components/pagination/pagination.component.html]: shouldn't this be:
```
*ngIf="totalPages > 1"
```

Comment (jacobwindsor) [reply src/app/projects/components/pagination/pagination.component.html]: Wow brain not working...

**Review by aaclan-ebi — APPROVED** [review]

**Review by MightyAx — APPROVED** [review]

**Pull request merged**